src/test/java/com/wikia/webdriver/common/clicktracking/events/EventsGlobalNavigation.java

```diff
@@ -0,0 +1,88 @@
+package com.wikia.webdriver.common.clicktracking.events;
```

Comment:
good idea but it would be part of refactoring to clicktracking system
created: https://wikia-inc.atlassian.net/browse/QAART-547